src/multibody/actions/free-fwddyn.cpp

```diff
@@ -69,7 +69,6 @@ void DifferentialActionModelFreeFwdDynamics::calc(const boost::shared_ptr<Differ
   }
 
   // Computing the cost value and residuals
-  pinocchio::updateFramePlacements(pinocchio_, d->pinocchio);
```

Comment:
Could we replicate this on `contact-fwddyn.cpp`?

Reply:
I'm on the calcdiff right now :) , afterwards I'll look at contactDAM.

Reply:
Great! thanks!
Do not forget to share the results.